### ISSUE TITLE
UI: Add Thanos Logo in the navbar

### ISSUE THREAD

**Issue by Nexucis** [thanos-io/thanos#5121]
It would be cool to have the Thanos logo in the top navbar of the UI.
It doesn't look like the logo is somewhere in the UI and displaying in the top navbar is pretty common.

**Comment by Harshitavkb26** [comment]
Hi.. I would like to work on this ,can you assign it to me?

**Comment by Nexucis** [comment]
well I'm not part of Thanos, but I'm pretty sure you can send a PR without having the issue assigned to you :).

**Comment by Harshitavkb26** [comment]
> well I'm not part of Thanos, but I'm pretty sure you can send a PR without having the issue assigned to you :).

okay  ✌️

**Comment by tend2infinity** [comment]
I'll fix this

**Comment by Nexucis** [comment]
We are waiting the merge of the associated PR. Feature is still accurate

**Comment by Nexucis** [comment]
up

**Comment by hackeramitkumar** [comment]
@GiedriusS I would like to work on it.

**Comment by Nexucis** [comment]
there is an open PR here: https://github.com/thanos-io/thanos/pull/5150. But it looks like it's a bit outdated

**Comment by hackeramitkumar** [comment]
@Nexucis Yeah I have seen that but there is no progress on that. So I was thinking to open a new PR.

**Comment by Nexucis** [comment]
yeah probably it's ok to do it :)
